spec/controllers/packages_controller_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'rails_helper'
+require 'support/vcr_setup'
+require 'rubygems'
+
+RSpec.describe PackagesController, type: :controller do
+
+  describe "POST rates" do
+    let(:json) { JSON.parse(response.body) }
+
+    context "valid params" do
+      let(:params) {
+        { zip: "98101", city: "Seattle", state: "WA", country: "US", box_size: "medium", total_weight: "12.0", controller: "packages", action: "estimate_request"}
+        }
+
+      before :each do
+        VCR.use_cassette 'delete' do
+          get :estimate_request, params
+        end
+      end
+
+      it "is successful" do
+        expect(response.response_code).to eq(200)
+      end
+
+      it "returns json" do
+        expect(response.header['Content-Type']).to include 'application/json'
+      end
+
+      it "is an array of arrays" do
+        expect(json).to be_an_instance_of Array
+      end
+    end
+  end
+
+  describe "changes box size to dimensions" do
+    it "changes the box size received form bEtsy to dimensions that can be comsumed by the shipping API gem" do
+      @controller = PackagesController.new
+      @box_size = "small"
+      @controller.instance_eval{ set_box_size }
+      expect(@controller.instance_eval{ set_box_size }).to eq([8, 8, 8])
```

Comment:
As I mentioned in the controller, making this method take and return a value would be better, and easier to test.

You appear to be calling the method twice here. Do you need line 39?

Instead of `instance_eval` you could just use `@controller.send(:set_box_size)`.

Also, I belive in a controller spec there's a method `controller` that returns the controller instance for testing, so you don't need to instantiate a new one.